webapp/sources/rudder/rudder-web/src/main/elm/editor/sources/ViewBlock.elm

```diff
@@ -207,14 +205,15 @@ showBlockTab model parentId block uiInfo techniqueUi =
       let
         compositionText  = (\reportingLogic ->
                                case reportingLogic of
-                                 WorstReport -> "Worst report"
-                                 SumReport -> "Sum of reports"
+                                 WorstReportWeightedSum -> "Worst report (weighted sum)"
```

Comment:
Should only be one entry with Worst report, and a second select to chose the weight, default all

Reply:
That would make two clicks, which is worse than one no? Or a "select with sub-select opening on mouseover the choice" ? (but I don't know how to do that)

Reply:
The cost of understanding what the two option means and in what they differ is costier to me

Reply:
To me it's easier to understand if there is less option, and a to think about the wieght after